src/ralph/scan/plugins/ssh_cisco_asa.py

```diff
@@ -110,7 +110,8 @@ def scan_address(ip_address, **kwargs):
         ssh.close()
     pairs = parse.pairs(lines=[line.strip() for line in lines])
     sn = pairs.get('Serial Number', None)
-    model, ram, cpu = pairs['Hardware'].split(',')
+    # hardware = model, ram, cpu, corenum*
+    hardware = pairs['Hardware'].split(',')
```

Comment:
so maybe sth like this:
```
model, ram, cpu = pairs['Hardware'].split(',')[:3]
```
this will work even if in some new devices they'll add another variable and we'll keep nice variables naming instead of `hardware[0]` etc.